Homework-1/index.js

```diff
@@ -83,9 +85,24 @@ function calculateAverage(reviews) {
 function handleStarClick(index) {
   const stars = document.getElementsByClassName("star");
   for (let i = 0; i < 5; i++) {
-    if (i <= index)
-      document.getElementsByClassName("star")[i].style.color = "yellow";
-    else document.getElementsByClassName("star")[i].style.color = "white";
+    if (i <= index) stars[i].classList = "fas fa-star star star-clicked";
+    else stars[i].classList = "fas fa-star star";
   }
   document.getElementById("rating").value = index + 1;
 }
+
+function handleHover(index) {
+  const stars = document.getElementsByClassName("star");
+  for (let i = 0; i < 5; i++) {
+    if (i <= index && !stars[i].classList.contains("star-clicked"))
```

Comment:
Not sure if I already commented on this, but I like to avoid writing if blocks without curly braces. I think it makes it more readable and less error prone.
It is a personal preference though, so if you really love it, go for it.